frontend/ui/data/koptoptions.lua

```diff
@@ -259,6 +260,16 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
                 },
                 alternate = false,
                 values = {7, 6, 5, 4, 3, 2, 1, 0},
+                labels = {
+                    _("Left to Right, Top to Bottom"),
```

Comment:
FWIW, CamelCase was voluntary here.

(One, it helped me make sure I didn't get it wrong, two, I kind of like it? ^^).